src/Pdf.php

```diff
@@ -215,4 +215,24 @@ protected function determineOutputFormat(string $pathToImage): string
 
         return $outputFormat;
     }
+
+    private function fetchNumberPages(string $pdfFile)
+    {
+        $stream = fopen($pdfFile, 'r');
+        $content = fread($stream, filesize($pdfFile));
```

Comment:
I don't understand how you can determine the number of pages using `file size`

Reply:
Not all PDF files contain "Count". Accordingly this method don't work correctly on some files.

Reply:
Right...